src/script/auth/page/Login.tsx

```diff
@@ -163,6 +174,16 @@ const Login = ({
             history.push(ROUTE.CLIENTS);
             break;
           }
+          case BackendError.LABEL.CODE_AUTHENTICATION_REQUIRED: {
```

Comment:
keep in mind theres also a FE Feature flag for this, so possibly you would want to check if it is enabled. 

``# Feature toggle for 2nd factor authentication for some actions (ex. login)
FEATURE_ENABLE_2F_PASSWORD_CHALLENGE="false"``